Dockerfile

```
@@ -15,13 +15,13 @@
 #
 
 # -- Base image --
-FROM python:3.8-slim as base
+FROM python:3.8-slim AS base
 
 # Upgrade pip to its latest release to speed up dependencies installation
 RUN pip install --upgrade pip
 
 # ---- Front-end builder image ----
-FROM node:14 as front-builder
+FROM node:14 AS front-builder
```

You Upgrade the node's image in the CI definition and the compose file. 
Maybe here too ?
```suggestion
FROM node:16 AS front-builder
```